packages/utilities/src/dom/on.ts

```diff
@@ -1,7 +1,9 @@
+import * as React from 'react';
+
 export function on(
   element: Element | Window,
   eventName: string,
-  callback: (ev: Event) => void,
+  callback: (ev: Event | React.SyntheticEvent<Element, Event>) => void,
```

Comment:
I don't think this is the right change--since the listener is added using `addEventListener`, it will always be a real event, not a React event. So if `callback` (as authored by `on`'s caller) was previously written to only expect React events, its types need to be updated to also handle real events.

Reply:
Very good point, I just submitted https://github.com/microsoft/fluentui/pull/12753 to revert this change and update ColorRectangle and ColorSlider to handle native events.